internal/app/connectconformance/testsuites/data/connect_client_error_endstream.yaml

```diff
@@ -83,6 +83,25 @@ testCases:
       error:
         code: CODE_UNAVAILABLE
         message: oops
+  - request:
+      testName: error/message-omitted
```

Comment:
For consistency with above test case (named `missing-code`), how about `missing-message` instead?

Reply:
Code is required, but message is optional, so it isn't really missing. Maybe that's splitting hairs... I've renamed to `missing-message` for consistency, and added a comment explaining it's optional.